.rubocop_todo.yml

```diff
@@ -10,7 +10,7 @@
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
-  Max: 180
+  Max: 200
```

Comment:
you can run `rubocop -a` to generate a new one instead of hand-editing

Reply:
I removed this block and added the spec file to the exclusion list for just this rule, which should avoid subsequent issues with new spec tests.